EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -72,8 +79,28 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
     private func onLoad() {
         play()
         observeMediaFinished()
+        addPeriodicTimeObserver()
     }
-    
+
+
+    private func addPeriodicTimeObserver() {
+        let intervalInMilliseconds = Int64(Constants.barUpdateIntervalInMs)
```

Comment:
Why are you storing `barUpdateIntervalInMs` as a `CGFloat` if you're casting it to an `Int` here anyway?